File: JobRecruitment/src/main/java/bachelor/thesis/job_recruitment/core/model/Location.java
package bachelor.thesis.job_recruitment.core.model;

import lombok.*;

import javax.persistence.Entity;
import javax.persistence.OneToOne;

@EqualsAndHashCode(callSuper = true)
@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Builder
public class Location extends BaseEntity<Long>{
    private Double latitude;
    private Double longitude;
}

File: JobRecruitment_Frontend/src/app/model/Location.ts
import {BaseEntity} from "./BaseEntity";
import {User} from "./User";

export class Location extends BaseEntity{
  latitude: number;
  longitude: number;

  constructor(id: number, latitude: number, longitude: number) {
    super(id);
    this.latitude = latitude;
    this.longitude = longitude;
  }
}
